apps/core-angular-universal/package.json

```diff
@@ -4,53 +4,49 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build --prod && ng run core-angular-universal:server:production",
+    "build": "ng build && ng run core-angular-universal:server:production",
```

Comment:
Am I missing something? Where are the `core-angular-universal:*` scripts coming from?

Reply:
Its a convention for the ng cli I think. `ng run` takes a param for the build process thats listed in the `angular.json` file